package.json

```diff
@@ -38,9 +41,9 @@
     "snyk": "^1.40.2"
   },
   "devDependencies": {
-    "istanbul": "^0.4.5",
-    "mocha": "3",
-    "supertest": "^2.0.0"
+    "istanbul-harmony": "0",
+    "mocha": "^3.5.0",
+    "supertest": "^3.0.0"
```

Comment:
Sweet, thanks for fixing the supertest update. I also like that we're using more `^` versions here.